qpc/report/tests_report_merge.py

```diff
@@ -111,8 +111,8 @@ def test_detail_merge_job_ids(self):
                              json_dir=None)
             with redirect_stdout(report_out):
                 nac.main(args)
-                self.assertEqual(report_out.getvalue().strip(),
-                                 messages.REPORT_SUCCESSFULLY_MERGED % '1')
+                self.assertIn(messages.REPORT_SUCCESSFULLY_MERGED % ('1', '1'),
```

Comment:
can we not assert that these are equal instead of asserting that it is in the report_out.getvalue()?

Reply:
Fixed.